lib/srv/desktop/tdp/proto_test.go

```diff
@@ -200,3 +200,15 @@ func TestMFA(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, respWant, respGot)
 }
+
+func TestIsNonFatalErr(t *testing.T) {
```

Comment:
Test a fatal error too? You can generate a slice that's too large with `bytes.Repeat`.

Reply:
1d3851cddba0b9f236d4e410c2c9acd37dc3668f